examples/dataflow-flex-python/df-sample-xml-to-bq/metadata.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "XML to BQ Flex template",
+    "description": "dataflow flex template for batch ingestion of XML files to BigQuery or json files",
+    "parameters": [
+        {
+            "name": "input",
+            "label": "Source File Path",
+            "helpText": "Specify Source Path to folder containing the input XML files",
+            "isOptional": false
+        },
+        {
+            "name": "output",
+            "label": "BQ output table",
+            "helpText": "Specify BQ Output table(project:dataset.table) or json file(example.json)",
+            "isOptional": false,
+            "regexes": [
+                "([^:]+:)?[^.]+[.].+"
+            ]
```

Comment:
This works for BQ table identifiers, but excludes some path matches. I'd drop the regex entirely.

Reply:
Removed